src/lib/MongoQueue.ts

```diff
@@ -0,0 +1,94 @@
+import { Model, Schema } from 'mongoose';
+import { Queue, Media, QueueManager } from 'musicca';
+
+export interface IQueue {
+  _id: string;
+  list: Media[];
+}
+
+export const schema = new Schema<IQueue>({
+  _id: String,
+  list: {
+    type: [Object],
+    default: []
+  }
+});
```

Comment:
Suggestion: move this to be static property in the `MongoQueue` class so accessing it would be easier
```ts
import { MongoQueue } from '@musicca/structs';

const schema = MongoQueue.schema;
```